examples/prometheus/prometheus.yaml

```diff
@@ -66,6 +66,7 @@ objects:
   metadata:
     annotations:
       prometheus.io/scrape: "true"
+      prometheus.io/port: "8443"
```

Comment:
Shouldn't this be the port exposed by the service (443) vs the target port or does that depend on how prometheus is configured to scrape metrics?

Reply:
Since Prometheus is scraping the target from the "inside", it should be the target port (eg 8443).